app/views/planning_applications/neighbour_letter_batches/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "application/head" %>
+<div class="display-flex">
+  <div class="neighbour-letter-export">
+    <div class="govuk-!-text-align-right">[address]</div>
+    <br><br>
+    <%= @batch.created_at.to_date.to_fs %>
+    <br><br>
+    <%= @batch.text %>
```

Comment:
Is there a reason for the <br> tags and not using <div> wrappers and a govuk-!-margin-* class?